doc/configuration/realms.rst

```diff
@@ -18,6 +18,49 @@ Therefor the users need to authenticate with their username and the realm like t
    
    user@realm
 
+.. _relate_realm:
+
+Relate User to a Realm
+......................
+
+.. index:: realm relation
+
+There are several options to relate a user to a specific realm during
+authentication. Usually, if only a login name is given, the user will be
+searched in the default realm.
+
+If a *realm* parameter is given in a :ref:`/auth<rest_auth>` or
+:ref:`/validate/check<rest_validate>` request, it supersedes a possible
+:ref:`split<splitatsign>` realm.
+
+The following table shows different combinations of a users login and a *realm*
+parameter and where the user will be looked for depending on the
```

Comment:
Should it be "where a user will be looked up" or "where the user will be searched"?